config/cmodel.py

```diff
@@ -0,0 +1,9 @@
+# Enable CModel mags (unsetup meas_modelfit to disable)
+# 'config' is a SourceMeasurementConfig.
+try:
+    import lsst.meas.modelfit
+    config.measurement.plugins.names |= ["modelfit_DoubleShapeletPsfApprox", "modelfit_CModel"]
+    config.measurement.slots.modelFlux = 'modelfit_CModel'
+    config.catalogCalculation.plugins['base_ClassificationExtendedness'].fluxRatio = 0.985
+except (KeyError, ImportError):
+    print("Cannot import lsst.meas.modelfit: disabling CModel measurements")
```

Comment:
Can this be a more proper log message, or does that not work in config files?

Reply:
In theory this is all python code so the except could be

```python
except (...):
    import logging
    logging.getLogger("lsst.obs.decam.config").warning("Cannot import... disabling...")
```

Reply:
Granted obs_subaru and obs_lsst both use `print()` but maybe logging is better for those anyhow.

Reply:
Changed to:
```
except (KeyError, ImportError):
    import logging
    logging.getLogger("lsst.obs.decam.config").warning("Cannot import lsst.meas.modelfit: disabling CModel measurements")
```